modin/pandas/test/test_io.py

```diff
@@ -248,7 +248,7 @@ def _csv_file_maker(
                 "delimiter": delimiter,
                 "doublequote": doublequote,
                 "escapechar": escapechar,
-                "lineterminator": line_terminator,
+                "lineterminator": line_terminator if line_terminator else os.linesep,
```

Comment:
why do we replaced the default parameter of `line_terminator` to `None` if further we're replacing `None` to `os.linesep`?

Reply:
`line_terminator` is common parameter for pandas.to_csv and for csv.reader/writer (`lineterminator` for `csv` module).
In the `pandas` case default parameter value is None (corresponds to actual using of `os.linesep`), but for `csv` module None is not acceptable parameter value and we should provide exact value (`os.linesep`).

Reply:
ok, got it